src/screen.py

```diff
@@ -50,7 +51,10 @@ def convert_abs_to_monitor(self, abs_coord: Tuple[float, float]) -> Tuple[float,
         monitor_coord = self.convert_screen_to_monitor(screen_coord)
         return monitor_coord
 
-    def grab(self) -> np.ndarray:
+    def grab(self, hide_mouse=False) -> np.ndarray:
+        if hide_mouse:
+            mouse.move(0, 0)  # Moving the mouse to the edge of the screen to not cover any loot.
```

Comment:
You will have to convert 0, 0 to monitor coordinates. If someone is running a two monitor setup the (0, 0) screen coordinate might be at (1920, 0) for example.